apps/token/src/routes/staking/staking-node.tsx

```diff
@@ -0,0 +1,149 @@
+import './staking-node.scss';
+
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+
+import { EpochCountdown } from '../../components/epoch-countdown';
+import { Colors } from '../../config';
+import type { VegaKeyExtended } from '@vegaprotocol/wallet';
+import { BigNumber } from '../../lib/bignumber';
+import type { Staking as StakingQueryResult } from './__generated__/Staking';
+import { ConnectToVega } from './connect-to-vega';
+// import { PendingStake } from "./pending-stake";
```

Comment:
?

Reply:
This is already commented out in the old repo

Reply:
Deleted